gradle/gradle-mvn-push.gradle

```diff
@@ -42,6 +42,101 @@ def getRepositoryPassword() {
   return hasProperty('SONATYPE_NEXUS_PASSWORD') ? SONATYPE_NEXUS_PASSWORD : ""
 }
 
+def kotlinAndroidArtifactTasks() {
+  if (!project.plugins.hasPlugin('org.jetbrains.dokka-android')) {
+    throw new GradleException("Apply the dokka-android plugin in ${project.name}")
+  }
+
+  dokka {
+    externalDocumentationLink {
+      url = new URL("http://reactivex.io/RxJava/2.x/javadoc/")
```

Comment:
should this point to https://uber.github.io/AutoDispose/0.x/ ?

Reply:
Hm, what are you thinking here? That's what builds in linking between the kdoc and rxjava's doc, so if you're looking at documentation that links to rxjava you can click on it and jump straight to the rxjava documentation. 

I could see adding https://uber.github.io/AutoDispose/0.x/ being useful (at least until there's proper inter-module linking). Although I think each module's doc might need to be linked independently. The dokka examples indicate you can have multiple external documentation links so if you want I could add it and keep rxjava.

I actually didn't even realize the documentation was hosted at  https://uber.github.io/AutoDispose/0.x/, this PR just creates javadoc artifacts for maven. 

I see https://github.com/uber/AutoDispose/tree/gh-pages/0.x but don't really know how/if the process of getting the doc in there is automated. If you can point me in the right direction I'd be happy to include that as part of this PR.

Reply:
yeah I'd held off on publicizing the docs site till dokka was figured out. I upload them via [osstrich](https://github.com/square/osstrich), unfortunately not sure automating it is viable vs just doing it manually after releases (don't worry about it for this PR). I didn't realize that was for dependencies of the project though, so this makes sense :+1: